packages/next/build/webpack/config/blocks/css/loaders/global.ts

```diff
@@ -28,8 +28,11 @@ export function getGlobalCssLoader(
     options: {
       importLoaders: 1 + preProcessors.length,
       sourceMap: true,
-      // Next.js controls CSS Modules eligibility:
-      modules: false,
+      // don't use `modules: false` or else syntax errors won't
+      // be built correctly
+      modules: {
+        mode: 'global',
+      },
```

Comment:
This will slow down compile time. Can we try to fix this while leaving `modules: false`?